presto-main/src/main/java/com/facebook/presto/operator/PageProcessor.java

```diff
@@ -24,13 +23,9 @@
 
 public interface PageProcessor
 {
-    int process(ConnectorSession session, Page page, int start, int end, PageBuilder pageBuilder);
-
     /**
      * @return filtered and projected page, null if all positions are filtered
      */
     @Nullable
-    Page processColumnar(ConnectorSession session, Page page, List<? extends Type> types);
-
-    Page processColumnarDictionary(ConnectorSession session, Page page, List<? extends Type> types);
+    Page process(ConnectorSession session, Page page, List<? extends Type> types);
```

Comment:
You are adding `@Nullable` for new `proces`s method which actually is mapped to old `processColumnarDictionary`.
Is the fact old method was not annotated just omission in old code?

Edit: Oh I see it was.